### ISSUE TITLE
#30996 Sales Order Acknowledgement Print, lines overwrite header

### ISSUE THREAD

**Issue by steveblamey** [uzerpllp/uzerp#1]

**Comment by steveblamey** [comment]
Example SO Acknowledgement showing the issue

**Comment by steveblamey** [comment]
I've tracked this down to a problem in the XSL that generates the document. A 10cm margin is required to accomodate the header on each page but the XSL specifies 7cm for the first page of the document.

```
				<!-- layout for the all pages -->
				<fo:simple-page-master master-name="rest"
						page-height="29.7cm"
						page-width="21cm"
			                        margin="0.5cm" >
					<fo:region-body margin-top="7cm" margin-bottom="7mm"/>
					<fo:region-before extent="7cm"  />
				</fo:simple-page-master>
				<!-- layout for the last pages -->
				<fo:simple-page-master master-name="last"
						page-height="29.7cm"
						page-width="21cm"
			                        margin="0.5cm" >
					<fo:region-body margin-top="10cm" margin-bottom="7mm"/>

					<fo:region-before extent="10cm"  />
					<fo:region-after extent="5mm"/>
				</fo:simple-page-master>
```

**Comment by steveblamey** [comment]
fixes #1

Add phinx migration (data only).
Adjust top margin on Sales Order Acknowledgement document.


→ <<cset 2b444deeb3d6>>
